getting_started/access_your_services.adoc

```diff
@@ -59,7 +59,46 @@ https://logs.<cluster-id>.openshift.com
 ----
 https://api.<cluster-id>.openshift.com/oauth2callback/<IdP name>
 ----
+endif::[]
+
+ifdef::openshift-arho[]
+[[access-your-cluster-overview]]
+== Access your cluster
+
+// TODO Jim Minter is to provide updated URL info for ARHO
+
+Once your {product-title} cluster is configured and ready to use, you can access
+it through the following paths:
+
+* *Cluster ID*: The unique cluster name provided by the customer during provisioning. It is lowercase, and only contains letters, numbers, and hyphens.
+
```

Comment:
* *Cluster Region*: The region in which the resources have been create in. Current list includes :  australiaeast, canadacentral, canadaeast, eastus, westus, westeurope, northeurope.

Reply:
Don't list the regions in the documentation as they will change all the time